### PR TITLE
Prepare release of ghcide 0.2.0

### PR DESCRIPTION
**Pull request by cocreature**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 ### unreleased
 
+### 0.2.0 (2020-06-02)
+
+* Multi-component support (thanks @mpickering)
+* Support for GHC 8.10 (thanks @sheaf)
```
Comment (pepeiborra) [inline CHANGELOG.md:6]: this was contributed by @chshersh iirc

File: ghcide.cabal [hunk 2]
```diff
@@ -2,12 +2,12 @@ cabal-version:      1.20
...
 category:           Development
 name:               ghcide
-version:            0.1.0
+version:            0.2.0
 license:            Apache-2.0
 license-file:       LICENSE
 author:             Digital Asset
 maintainer:         Digital Asset
-copyright:          Digital Asset 2018-2019
+copyright:          Digital Asset 2018-2010
```
Comment (pepeiborra) [inline ghcide.cabal:10]: you probably meant 2020?

Comment (cocreature) [reply ghcide.cabal]: :facepalm:

File: CHANGELOG.md (hunk 1 above)
Comment (cocreature) [reply CHANGELOG.md]: The first PR towards 8.10 support was https://github.com/digital-asset/ghcide/pull/360. I’ve added @chshersh as well now.

**Pull request merged**